docs/docs/reference/05-oauth-providers/whop.md

```diff
@@ -0,0 +1,34 @@
+---
+id: whop
+title: Whop
+---
+
+## Documentation
+
+http://dev.whop.com
+
+## Configuration
+
+https://dash.whop.com/settings/developer
+
+## Options
+
+The **Whop Provider** comes with a set of default options:
+
+- [Whop Provider options](https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/whop.ts)
+
+You can override any of the options to suit your own use case.
+
+## Example
```

Comment:
Add the Vercel template here!

https://vercel.com/templates/next.js/whop-template